rust/psibase/src/boot.rs

```diff
@@ -52,6 +52,7 @@ const ACCOUNTS: [AccountNumber; 27] = [
     account!("token-sys"),
     transaction_sys::SERVICE,
     account!("verifyec-sys"),
+    account!("verify-sys"),
```

Comment:
The names are a bit misleading now, since nist curves are ec, but can only be verified with verify-sys. Maybe rename verifyec-sys to verifyk1-sys?